python/py_src/sudachipy/command_line.py

```diff
@@ -51,26 +51,37 @@ def _set_default_subparser(self, name, args=None):
 argparse.ArgumentParser.set_default_subparser = _set_default_subparser
 
 
-def run(tokenizer, mode, input_, output, logger, print_all, enable_dump):
+def run(tokenizer, input_, output, print_all, morphs, is_stdout):
     for line in input_:
         line = line.rstrip('\n')
         # Note: Current version of the tokenizer ignores logger
-        for m in tokenizer.tokenize(line, mode, logger if enable_dump else None):
-            list_info = [
-                m.surface(),
-                ",".join(m.part_of_speech()),
-                m.normalized_form()]
+        mlist = tokenizer.tokenize("")
```

Comment:
It would be nice to have a comment about reuse technique of morpheme list

Reply:
Super nice catch, I completely missed this one